src/Uno.UI/Controls/Window.iOS.cs

```diff
@@ -252,7 +257,7 @@ public override UIView HitTest(CGPoint point, UIEvent uievent)
 					// In this case we have to send back the currently focused view in order to prevent EndEditing which will cause
 					// the keyboard to disappear and the feeling that touches are going "through the keyboard".
 
-					return _focusedView;
+					return _focusedView ?? base.HitTest(point, uievent);
```

Comment:
@MartinZikmund could you please validate this is Ok?

cannot return `null` from this method but `_focusedView` could be null at some point. I see this is the same logic followed in other places in this file but want to validate.

Reply:
I am not sure here, as per the comment above this line - it sounds like this could close the keyboard 🤔 . Why can't this method return `null`? Could we handle it in the caller potentially?

Reply:
we cannot return null from this method because this would break the method signature (it's an override)

Reply:
cant you force it with `_focusedView!`?

Reply:
True, i didn't consider that. Thanks